src/applications/vaos/tests/appointment-list/components/PastAppointmentsListV2.unit.spec.jsx

```diff
@@ -0,0 +1,321 @@
+  });
+
+  // Skipped until cancelled functionality added
+  it.skip('should have correct status when previously cancelled', async () => {
+    const appointment = getVAAppointmentMock();
+    appointment.attributes = {
+      ...appointment.attributes,
+      startDate: pastDate.format(),
+      clinicFriendlyName: 'Some clinic',
+      facilityId: '983',
+      sta6aid: '983GC',
+    };
+    appointment.attributes.vdsAppointments[0].currentStatus =
+      'CANCELLED BY PATIENT';
+    mockPastAppointmentInfo({ va: [appointment] });
+
+    const screen = renderWithStoreAndRouter(<PastAppointmentsListV2 />, {
+      initialState,
+      reducers,
+    });
+
+    await screen.findByText(
+      new RegExp(pastDate.tz('America/Denver').format('MMMM YYYY'), 'i'),
+    );
+
+    screen.debug();
+
+    const firstCard = screen.getAllByRole('listitem')[0];
+
+    expect(within(firstCard)).to.contain.text('Canceled');
+    expect(within(firstCard)).to.contain('.fa-exclamation-circle');
+    expect(within(firstCard)).not.to.contain.text('Add to calendar');
+    expect(within(firstCard)).not.to.contain.text('Cancel appointment');
+  });
+
+  it('should not display when they have hidden statuses', () => {
+    const appointment = getVAAppointmentMock();
+    appointment.attributes.startDate = moment().format();
+    appointment.attributes.vdsAppointments[0].currentStatus = 'NO-SHOW';
+
+    mockPastAppointmentInfo({ va: [appointment] });
+    const screen = renderWithStoreAndRouter(<PastAppointmentsListV2 />, {
+      initialState,
+      reducers,
+    });
+
+    return expect(screen.findByText(/You don’t have any appointments/i)).to
+      .eventually.be.ok;
+  });
+
+  it('should not display when over 13 months away', () => {
+    const appointment = getVAAppointmentMock();
+    appointment.attributes.startDate = moment()
+      .add(14, 'months')
+      .format();
+    appointment.attributes.vdsAppointments[0].currentStatus = 'FUTURE';
+
+    mockPastAppointmentInfo({ va: [appointment] });
+    const screen = renderWithStoreAndRouter(<PastAppointmentsListV2 />, {
+      initialState,
+      reducers,
+    });
+
+    return expect(screen.findByText(/You don’t have any appointments/i)).to
+      .eventually.be.ok;
+  });
+
+  it('should show expected video information', async () => {
+    const appointment = getVideoAppointmentMock();
+    appointment.attributes = {
+      ...appointment.attributes,
+      facilityId: '983',
+      clinicId: null,
+      startDate: pastDate.format(),
+    };
+    appointment.attributes.vvsAppointments[0] = {
+      ...appointment.attributes.vvsAppointments[0],
+      dateTime: pastDate.format(),
+      bookingNotes: 'Bring face mask',
+      status: { description: 'C', code: 'CHECKED OUT' },
+    };
+    mockPastAppointmentInfo({ va: [appointment] });
+
+    const screen = renderWithStoreAndRouter(<PastAppointmentsListV2 />, {
+      initialState,
+      reducers,
```

Comment:
You can remove reducers here and for the rest of the tests in this file, it's not necessary. I'm trying to clean these up in our other tests as we make changes, but haven't gotten all of them.

Reply:
Got it, removed.